.changeset/funny-waves-worry.md

```diff
@@ -0,0 +1,17 @@
+---
+'astro': minor
+'@astrojs/netlify': minor
+---
+
+Introduces the **experimental** Prerender API.
+
+> **Note**
+> This API is not yet stable and is subject to possible breaking changes!
+
+- Deploy an Astro server without sacrificing the speed or cacheability of static HTML.
+- The Prerender API allows you to statically prerender specific `pages/` at build time.
+
+**Usage**
+
+- First, enable `experimental: { prerender: true }` in your `astro.config.mjs` file.
```

Comment:
you should also be able to enable this through a CLI flag, ala `--experimental-prerender`.

Reply:
Fixed in [9e4a190](https://github.com/withastro/astro/pull/5297/commits/9e4a19076d9fa6e803d440f543c386e940116e57)